pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -142,7 +142,7 @@ func checkIfDaemonTaskRunning() error {
 	if strings.TrimSpace(stdout) != "Running" {
 		return fmt.Errorf("expected %s task to be in 'Running' but got '%s'", constants.DaemonTaskName, stdout)
 	}
-	return nil
+	return checkIfDaemonIsServing()
```

Comment:
@praveenkumar I think this should be also called from the `fix` function, `fixDaemonTaskRunning` no?

Reply:
just noticed, its done in this way in the commit mentioned here: https://github.com/code-ready/crc/pull/3150#pullrequestreview-954479941